halibot/halobject.py

```diff
@@ -48,6 +53,20 @@ def send_to(self, msg, dests):
 				self.log.warning('Unknown module/agent: ' + str(name))
 		return ret
 
+	def sync_send_to(self, msg, dests):
+		msg.sync = True
+
+		futs = self.send_to(msg, dests)
+
+		r = {}
+		for name, fut in futs.items():
+			fut.result()
+			to = self._hal.objects.get(name)
+			if to and msg.uuid in to.sync_replies:
+				# Assure that the module was not removed in the interim
+				r[name] = to.sync_replies.pop(msg.uuid)
+		return r
```

Comment:
So this returns a map of `module -> replies`, correct?
Will there ever be a case that we might want to fire off the `sync_` message, do some other things, then come back to waiting on that first message?

In other words, would it be worth returning the future so we can `await sync_send_to`?